src/common/classes/ClumpletWriter.cpp

```diff
@@ -437,6 +440,9 @@ void ClumpletWriter::insertBytesLengthCheck(UCHAR tag, const void* bytes, const
 			cur_offset += 4;
 		}
 		break;
+	default:
+		invalid_structure("invalid length size", lenSize);
```

Comment:
lenSize is set only by our code, please remove this default label, use fb_assert() instead

Reply:
OK

Reply:
Right now I pushed a change with the second assert I mentioned in the switch; do you think that it's OK, or do you want it moved outside the switch, or as `fb_assert(false)`?